staging/src/k8s.io/client-go/tools/clientcmd/overrides.go

```diff
@@ -29,21 +29,23 @@ import (
 type ConfigOverrides struct {
 	AuthInfo clientcmdapi.AuthInfo
 	// ClusterDefaults are applied before the configured cluster info is loaded.
-	ClusterDefaults clientcmdapi.Cluster
-	ClusterInfo     clientcmdapi.Cluster
-	Context         clientcmdapi.Context
-	CurrentContext  string
-	Timeout         string
+	ClusterDefaults      clientcmdapi.Cluster
+	ClusterInfo          clientcmdapi.Cluster
+	Context              clientcmdapi.Context
+	CurrentContext       string
+	Timeout              string
+	IncludeUninitialized bool
```

Comment:
The config here is not specific for kubectl, so we shouldn't change it. Probably you should add it in [helpers.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/util/helpers.go#L425-L428) like other flags.